### ISSUE TITLE
tydux-angular references older tydux version

### ISSUE THREAD

**Issue by pburgmer** [w11k/Tydux#30]
npm install with dependencies

```
"@w11k/tydux": "^13.5.6",
"@w11k/tydux-angular": "^13.5.6",
``` 

prints

```
@w11k/tydux-angular@13.5.6 requires a peer of @w11k/tydux@12.x.x
```

**Comment by romanroe** [comment]
fixed in v13.5.7

**Comment by pburgmer** [comment]
Thanks
